sdk/search/search-documents/README.md

```diff
@@ -833,6 +899,49 @@ async function main() {
 main();
 ```
 
+#### Get the service statistics
+```js
+const { SearchServiceClient, AzureKeyCredential } = require("@azure/search-documents");
+
+const client = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
+
+async function main() {
+  const serviceStatistics = await client.getServiceStatistics();
```

Comment:
Can destructure the two properties here to avoid having to type `serviceStatistics` over and over

```suggestion
  const {counters, limits } = await client.getServiceStatistics();
```

Reply:
Done. Will update in the next commit